cuda/test/matrix/csr_kernels.cpp

```diff
@@ -30,7 +30,6 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ******************************<GINKGO LICENSE>*******************************/
 
-#include <ginkgo/core/matrix/coo.hpp>
 #include <ginkgo/core/matrix/csr.hpp>
```

Comment:
If we keep the same format as other tests (e.g. `dense`), here shouldn't this be:
```c++
#include "core/matrix/csr_kernels.hpp"
```

Reply:
Yes, I will change that.